### PR TITLE
implementing changes to boxplot and altair tests changed

### PR DESCRIPTION
**Pull request by carlinakim**

### CODE REVIEW

File: pysketball/nba_boxplot.py [hunk 1]
```diff
@@ -68,8 +69,8 @@ def nba_boxplot(dataset, stats, position=None, teams=None):
         raise TypeError("teams must be a list")
 
     # Checks if the position exists in the dataframe
-    if position is not None and position not in dataset.columns:
-        raise TypeError(f"Column {position} not found in data")
+    if position is not None and position != 'POS':
```
Comment (vanandsh) [inline pysketball/nba_boxplot.py]:
```suggestion
    if position is not None and position is not 'POS':
```

**Review by vanandsh — CHANGES_REQUESTED** [review]
Hi @carlinakim, I've added a suggestion. Please accept it if you feel it won't break anything.

File: pysketball/nba_boxplot.py (hunk 1 above)
Comment (carlinakim) [reply pysketball/nba_boxplot.py]: sounds good added!

**Comment by AndresPitta** [comment]
Hi @carlinakim,

I see your PR contains the following changes:

- Changed `test_nba_ranking` to not depend on V4 objects, addressing @kvarada's feedback in issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-602124775):

> Tests
> When I ran your tests, two tests failed with the following error.
> AttributeError: module 'altair.vegalite' has no attribute 'v4'

- Changed `nba_boxplot` to address @zoepan00's feedback in issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-601541684):

> Error message helps for the case if both position and teams have values. I think it would be nice to also include an error message if I input other columns names to the position argument(for example an error message for position="TEAM"), or change the position argument type to boolean.

Thanks,
Andres

**Pull request merged**